doc/source/ray-air/package-ref.rst

```diff
@@ -74,6 +74,15 @@ Feature Scalers
 .. autoclass:: ray.data.preprocessors.StandardScaler
     :show-inheritance:
 
+K-Bins Discretizers
```

Comment:
Would a more-general `Discretizers` be better? I see sklearn does something similar in their user guide https://scikit-learn.org/stable/modules/preprocessing.html#discretization

Reply:
Sure, makes sense!